Index: src/main/java/org/example/Fussballapplikation.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/Fussballapplikation.java b/src/main/java/org/example/Fussballapplikation.java
--- a/src/main/java/org/example/Fussballapplikation.java
+++ b/src/main/java/org/example/Fussballapplikation.java	(date 1768746191127)
@@ -2,25 +2,18 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 
 @SpringBootApplication
-@RequestMapping("/api/test")
-@RestController
 public class Fussballapplikation {
 
     public static void main(String[] args) {
-        SpringApplication.run(Fussballapplikation.class, args);
-    }
+        SpringApplication.run(Fussballapplikation.class, args);}
+        @GetMapping("/greetings")
+        public ResponseEntity<String> sayHelloWorld(@RequestParam(defaultValue= "World") String name) {
+            return ResponseEntity.ok("Hello " + name);
 
-    @PostMapping
-    public String testPost() {
-        return "POST works!";
-    }
+        }
 
 }
-
-
-
-
-
